src/Private/OpenApi.ps1

```diff
@@ -168,7 +168,7 @@ The Name of the ComponentSchemaJson reference.
 #>
 
 
-function Test-PodeOAComponentSchemaJson {
+function Test-PodeOAComponentchemaJson {
```

Comment:
The "S" for schema looks to have been dropped :D should be: `Test-PodeOAComponentSchemaJson`

Reply:
opps :) 
it's an internal function and it's not used. Maybe I should drop it